src/Zygote.jl

```diff
@@ -4,9 +4,12 @@ using LinearAlgebra, Statistics
 using LinearAlgebra: copytri!, AbstractTriangular
 using ArrayLayouts: MemoryLayout, AbstractColumnMajor
 
-import ZygoteRules: @adjoint, @adjoint!, AContext, adjoint, _pullback, pullback, literal_getproperty
+import ZygoteRules: @adjoint, @adjoint!, AContext, adjoint, _pullback, pullback
+import ZygoteRules: literal_getproperty, differential2legacy, legacy2differential
+import ZygoteRules: legacytype_warn
+import ZygoteRules
```

Comment:
Unless you're extending `differential2legacy`, `legacy2differential`, and `legacytype_warn`, they should probably be `using`ed rather than `import`ed. Same for `ZygoteRules` -- not sure that you need to `import` it rather than just `using` it.

Reply:
done